samples/Samples.HttpMessageHandler/Samples.HttpMessageHandler.csproj

```diff
@@ -2,6 +2,10 @@
   <PropertyGroup>
     <LoadManagedProfilerFromProfilerDirectory>true</LoadManagedProfilerFromProfilerDirectory>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="HttpClientHelpers.cs" />
+    <Compile Remove="WebClientHelpers.cs" />
```

Comment:
Why are these files in the project if we're not going to compile them?

Reply:
They weren't yet deleted from the directory. Removing.